waku/v2/protocol/filter/client.go

```diff
@@ -128,23 +131,27 @@ func (wf *WakuFilterLightNode) onRequest(ctx context.Context) func(s network.Str
 			wf.metrics.RecordError(decodeRPCFailure)
 			return
 		}
+		pubSubTopic := messagePush.PubsubTopic
 		//For now returning failure, this will get addressed with autosharding changes for filter.
 		if messagePush.PubsubTopic == nil {
-			logger.Error("empty pubsub topic")
-			wf.metrics.RecordError(decodeRPCFailure)
-			return
+			*pubSubTopic, err = getPubSubTopicFromContentTopic(messagePush.WakuMessage.ContentTopic)
+			if err != nil {
+				logger.Error("could not derive pubSubTopic from contentTopic", zap.Error(err))
```

Comment:
Should we add the contentTopic causing the error to the log?

Reply:
That is alreadu included in the errorString returned from getPubSubTopicFromContentTopic. Hence not needed to add here.